runtime/doc/options.txt

```diff
@@ -5160,10 +5160,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	security reasons.
 
 						*'shellcmdflag'* *'shcf'*
-'shellcmdflag' 'shcf'	string	(default: "-c"; Windows: "/c")
+'shellcmdflag' 'shcf'	string	(default: "-c"; Windows: "/s /c")
 			global
 	Flag passed to the shell to execute "!" and ":!" commands; e.g.,
-	"bash.exe -c ls" or "cmd.exe /c dir".  For Windows
+	"bash.exe -c ls" or "cmd.exe /s /c "dir"".  For Windows
```

Comment:
I guess backticks should be used here, it unusual for documentation to have nested unescaped quotes.

    `bash.exe -c ls` or `cmd.exe /s /c "dir"`.  For Windows